src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,167 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static RegistrationService registration;
+
+    @BeforeAll
+    static void beforeAll() {
+        registration = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_additionUsers_ok() {
+        User firstUser = new User();
+        firstUser.setLogin("Login");
+        firstUser.setPassword("passWord1");
+        firstUser.setAge(MIN_AGE);
+
+        User secondUser = new User();
+        secondUser.setLogin("login");
+        secondUser.setPassword("passWord1");
+        secondUser.setAge(MIN_AGE);
+
+        boolean expect = firstUser.equals(registration.register(firstUser));
+        assertTrue(expect);
+        expect = secondUser.equals(registration.register(secondUser));
+        assertTrue(expect);
+    }
+
+    @Test
+    void register_moreThanMinAge_Ok() {
+        User firstUser = new User();
+        firstUser.setLogin("Login");
+        firstUser.setPassword("passWord1");
+        firstUser.setAge(MIN_AGE + 1);
+
+        boolean expect = firstUser.equals(registration.register(firstUser));
+        assertTrue(expect);
+    }
```

Comment:
```suggestion
    @Test
    void register_moreThanMinAge_Ok() {
        User user = new User();
        user.setLogin("Login");
        user.setPassword("passWord1");
        user.setAge(MIN_AGE + 1);
        boolean expected = user.equals(registration.register(user));
        assertTrue(expected);
    }
```
if there is only one user, don't call it first user, because there is no second one
remove redundant empty line, they take space and your tests are short enough to be read easily
make changes like this in all tests